cwms-http-client/src/test/java/mil/army/usace/hec/cwms/http/client/TestHttpRequestResponse.java

```diff
@@ -25,24 +25,37 @@
 package mil.army.usace.hec.cwms.http.client;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNull;
 
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.stream.Collectors;
+import okhttp3.MediaType;
+import okhttp3.ResponseBody;
 import org.junit.jupiter.api.Test;
 
 class TestHttpRequestResponse {
 
     @Test
-    void testHttpRequestResponse() {
-        String body = "{\"message\":\"Unable to find category based on parameters given\",\"incidentIdentifier\":\"-49092149940938\",\"details\":{}}";
+    void testHttpRequestResponse() throws IOException {
+        String bodyStr =
+            "{\"message\":\"Unable to find category based on parameters given\",\"incidentIdentifier\":\"-49092149940938\",\"details\":{}}";
+        ResponseBody body = ResponseBody.create(bodyStr, MediaType.parse("application/json"));
         HttpRequestResponse httpRequestResponse = new HttpRequestResponse(body);
-        assertEquals(body, httpRequestResponse.getBody());
+        assertEquals(bodyStr, httpRequestResponse.getBody());
     }
 
     @Test
-    void testHttpRequestResponseNulls() {
-        String body = null;
+    void testHttpRequestResponseStream() {
```

Comment:
I'd really like to see a test with a binary file since that's how you are using the API.